doc/schemas/fix/codeclimate.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
can we have the reference where this file is coming from, linked?
so we can refer the original source, but also update the file if anything change in source file

Reply:
Sure, how do you suggest to link it?

Reply:
if possible in this file directly, then in this file.
if not, as a note in folder, (like codeclimate-ref.md)